_posts/2024-11-10-5.10.0-released.md

```diff
@@ -0,0 +1,111 @@
+future.
+
+## gettext Translations
+
+Luanti finally supports the standard [gettext](https://www.gnu.org/software/gettext/) `.po` and `.mo` translation file
+formats! These formats are much easier to manage via external tools such as Weblate, and also provide more options for
+translators (such as correct plural forms). The custom `.tr` translation file format is still supported, but we highly
+recommend migrating to gettext (see [Translations API](https://github.com/minetest/minetest/blob/5.10.0/doc/lua_api.md#translations))
+if you don't plan to support client versions older than 5.10.0.
+
+## Visual Effects
+
+Even more eye candy has arrived for players to enjoy! Among the list of options are simple liquid reflections and waves,
+translucent foliage, node specular highlights, softer clouds, and tinted shadows. This post's cover image features these
+effects.
+
+## Per-player Objects
+
+Developers can now control which player(s) an object is sent and shown to with
+[`object:set_observers()`](https://github.com/minetest/minetest/blob/5.10.0/doc/lua_api.md#methods-10). For example, the
+[Capture The Flag](https://content.luanti.org/packages/rubenwardy/capturetheflag/) server uses this for nametags which
+can only be seen by teammates. It could also be used to implement in-world interfaces or displays that only the desired
+player can see.
+
+This implementation is entirely server-side, so only the server needs to be 5.10.0 to support this feature. Clients do
+not need to upgrade to support this change.
+
+## More Highlights
+
+* **IPC**: The new [IPC API](https://github.com/minetest/minetest/blob/5.10.0/doc/lua_api.md#ipc) allows Luanti server
+  Lua environments to freely share data between all environments (server, async, and mapgen) via a key-value store,
+  making async and mapgen environments more powerful and easier to use.
+* **Bulk LBMs** let you run loading block modifiers on all the matching node positions in a mapblock at once with
+  [the `bulk_action` property](https://github.com/minetest/minetest/blob/5.10.0/doc/lua_api.md#lbm-loadingblockmodifier-definition).
+* [`get_player_control()`](https://github.com/minetest/minetest/blob/5.10.0/doc/lua_api.md#player-only-no-op-for-other-objects)
+  now exposes **joystick direction** (as X and Y axes). This may be especially useful for vehicles or arcade-inspired
+  games.
+* Mobile users will notice a grid menu in place of the button bar, as well as improved GUI scaling.
+* Server & content list search now requires all keywords to match.
+* Smooth scrolling can be toggled with a new setting.
+
+## Conclusion
+
+Check out [the full changelog](https://dev.minetest.net/Changelog#5.9.1_.E2.86.92_5.10.0) for other minor features and
+bugfixes. Development has resumed on 5.11.0, with work being done on [improving object
+structures](https://github.com/minetest/minetest/issues/14613), [custom touchscreen layouts](https://github.com/minetest/minetest/pull/14933),
```

Comment:
```suggestion
bugfixes. Development has resumed on 5.11.0, with work being done on [improving moving
object performance](https://github.com/minetest/minetest/issues/14613), [custom touchscreen layouts](https://github.com/minetest/minetest/pull/14933),
```

Reply:
To be pedantic it's not just about moving objects, it's more general than that, e.g. if you have many static items around that could also benefit (items check for nearby items to merge with), though moving objects (especially things looking for prey or something) are one major application.